### PR TITLE
[STARBUCKS-70]fix(lp):merge with v1.1.3 , update to v1.2.2-starbucks-oem

### PR DESCRIPTION
**Pull request by BlackBN**
**What this PR does / why we need it**:

fix(lp):merge with v1.1.3 , update to v1.2.2-starbucks-oem

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jianglinjian** [comment]
> [release-1.1][CPS-4825] feat: support concat multiline log (#70)  

这个 commit 不要 cherrypick 进去和星巴克OEM有冲突

File: build/filebeat-keeper/Dockerfile [hunk 1]
```diff
@@ -0,0 +1 @@
+FROM cargo.caicloud.xyz/library/filebeat:6.5.4
```
Comment (lichuan0620) [inline build/filebeat-keeper/Dockerfile:1]: 版本不对，星巴克用的 6.7.2

Comment (lichuan0620) [reply build/filebeat-keeper/Dockerfile:1]: 哦其实没事，这个 dockerfile 实际不会使用，是用来过流水线的 hack

**Comment by lichuan0620** [comment]
改用 #79
/close

**Pull request closed without merging**